Automattic/wp-calypso
# Update "Coming soon" features in cloud.jetpack.com/pricing modal

Related to #76992 

## Proposed Changes

* Update the style and copy of the "Coming soon" items in the pricing modal

<img width="1862" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/0a5ddb1b-e281-463c-ab71-59edf1fff278">


## Testing Instructions



* Visit the Jetpack Cloud live link
* Go to the pricing page /pricing
* Open the Jetpack Social product modal (by clicking on "[More about Social](http://jetpack.cloud.localhost:3001/pricing#jetpack_social_advanced_yearly)")
* Make sure the "Coming Soon" items appear at the top, it's bold, and that the copy matches what is described in #76992 
* Confirm it matches the screenshot above

## Pre-merge Checklist
